scale/scheduler/scale_scheduler.py

```diff
@@ -256,6 +256,8 @@ def statusUpdate(self, driver, status):
                     running_job_exe.task_running(task_id, results.when)
                 elif status.state == mesos_pb2.TASK_FINISHED:
                     running_job_exe.task_complete(results)
+                    if results.exit_code is None or not type(results.exit_code) is type(int):
+                        results.exit_code = 0
```

Comment:
This should be before `running_job_exe.task_complete(results)`

I don't think the int type check is really needed. Shouldn't it be "is not"?